homeassistant/components/scene/hunterdouglas_powerview.py

```diff
@@ -33,8 +33,8 @@ def setup_platform(hass, config, add_devices, discovery_info=None):
         _LOGGER.exception("error connecting to powerview "
                           "hub with ip address: %s", hub_address)
         return False
-    add_devices(PowerViewScene(hass, scene, _rooms, _pv)
-                for scene in _scenes['sceneData'])
+    add_devices([PowerViewScene(hass, scene, _rooms, _pv)
```

Comment:
This will not fix it. It is fine to put a generator expression without parentheses if the other positional arguments have a default value.

```
› python3
Python 3.5.2 (default, Oct 24 2016, 00:06:33)
[GCC 4.2.1 Compatible Apple LLVM 8.0.0 (clang-800.0.38)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> def hello(a, b=5):
...   print(list(a), b)
...
>>> hello(range(5))
[0, 1, 2, 3, 4] 5
>>> hello(i+1 for i in range(5))
[1, 2, 3, 4, 5] 5
```